test/Removal.test.ts

```diff
@@ -1,6 +1,6 @@
 import type { ContractInstances } from '@/test/helpers';
-import { deploy } from '@/deploy/0_deploy_contracts';
 import { expect, createFixture } from '@/test/helpers';
+import { deploy } from '@/deploy/0_deploy_contracts';
```

Comment:
is this used?

Reply:
yeah it's used in `setupTest` throughout the file

Reply:
I removed this in my PR.  I can help you adjust accordingly.